docs/developer/registry-access.md

```diff
@@ -104,30 +104,30 @@ response = registry._client.transport.receive()
 
 This is helpful for debugging situations where epplib is not correctly or fully parsing the XML returned from the registry.
 
-### Adding in a expiring soon domain
-The below scenario is if you are NOT in org model mode (`organization_feature` waffle flag is off).
+### Adding in an 'expiring soon' or 'expired' domain
+
+Expiration dates created by fixtures in sandboxes are not accurate (it resets to the expiration date to a year from 'today').
+In order to view an expiring or expired domain, we need to pull a record from staging and use it on the desired sandbox. In order to have edit ability, you must be a domain manager of that domain. Below are instructions on how to add a domain from staging sandbox into another sandbox and how to be able to view the domain. As a domain manager you will be able to view the renewal form.
```

Comment:
@lizpearl - can you explain how this works? I would expect the db to be totally separate between sandboxes. Why does creating (or getting?)  a domain to my sandbox with the same name as a domain in staging access the record?